lib/ansible/executor/module_common.py

```diff
@@ -204,7 +216,11 @@ def debug(command, zipped_mod, json_params):
     # Okay to use __file__ here because we're running from a kept file
     basedir = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'debug_dir')
     args_path = os.path.join(basedir, 'args')
-    script_path = os.path.join(basedir, 'ansible_module_%(ansible_module)s.py')
+    script_path = os.path.join(basedir, '__main__.py')
+
+    if command == 'excommunicate':
```

Comment:
Will this print mess things up?

Reply:
we already had similar warnings, the controller code will cut them as 'non json output from module'

Reply:
It shouldn't.  Because this is in the debug section and debug already prints other things to stdout.  Someone could be using debug commands in a way that isn't supported, though.  @nitzmahone, are you okay with this?  The two alternatives I can see are either, deprecate this in docs only (still removing in 2.11) or sending this to sys.stderr instead of stdout.